# config/config.yaml
waypoints:
  # - id: WP0                  # 필요하면 id 필드 추가
  #   lat: 35.069019
  #   lon: 128.086364
  #   alt: 0.0
  #   action: takeoff
  #   transition: null
  #   enabled: false           # enabled=false 면 코드에서 쉽게 필터링 가능

  # - id: WP1
  #   lat: 35.069019
  #   lon: 128.086364
  #   alt: 50.0
  #   enabled: false

  - id: WP1_5
    lat: 35.067994
    lon: 128.088238
    alt: 50.0
    transition: transition_fw   # 고정익 전환

  - id: WP2
    lat: 35.066984
    lon: 128.090191
    alt: 50.0
    action: loiter

  - id: WP3
    lat: 35.064587
    lon: 128.089794
    alt: 50.0
    action: loiter

  - id: WP4
    lat: 35.068178
    lon: 128.092905
    alt: 50.0
    action: loiter

  - id: WP5
    lat: 35.066984
    lon: 128.090191
    alt: 50.0
    action: loiter

  - id: WP5_5
    lat: 35.068758
    lon: 128.088163
    alt: 50.0
    transition: transition_mc   # 멀티콥터 전환

  - id: WP6
    lat: 35.069399
    lon: 128.086758
    alt: 50.0
    action: loiter

  - id: WP9
    lat: 35.068591
    lon: 128.085878
    alt: 50.0

  - id: WP12
    lat: 35.069019
    lon: 128.086364
    alt: 10.0
    action: land

parameters:
  initial_takeoff_altitude: 50.0 # meter
  final_altitude: 10.0 # meter
  altitude_increment: 10.0 # meter
  yaw_adjustment_duration: 5.0 # second
  loiter_radius: 30.0 # meter
  loiter_time: 10.0 # second
  threshold_range: 2.0 # meter
  threshold_height: 0.3 # meter
  ref_lat: 35.069019
  ref_lon: 128.086364


